authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
```

Comment:
Please resolve all conflicts before pushing for review

Reply:
resolved